core_debug.tex

```diff
@@ -133,7 +134,4 @@ \section{Core Debug Registers} \label{debreg}
 
 \section{Virtual Debug Registers} \label{virtreg}
 
-Virtual debug registers are a requirement on the debugger SW/interface,
```

Comment:
weren't people confused about this? I think we added this sentence to avoid confusion.

Reply:
Yes. I hope that this is covered by the beginning of sw_registers.xml, which I also rewrote in this change:
    A virtual register is one that doesn't exist directly in the hardware, but
    that the debugger exposes as if it does. Debug software should implement
    them, but hardware can skip this section. Virtual registers exist to give
    users access to functionality that's not part of standard debuggers without
    requiring them to carefully modify debug registers while the debugger is
    also accessing those same registers.

It's bit tricky to see that by reading the source, but this ends up exactly in the space of the line that I deleted there.

Reply:
Ah, I see it was added back below.